tests/load_balancer_test.go

```diff
@@ -0,0 +1,280 @@
+//
+// DISCLAIMER
+//
+// Copyright 2018 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+// Author Ewout Prangsma
+//
+
+package tests
+
+import (
+	"context"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/dchest/uniuri"
+
+	driver "github.com/arangodb/go-driver"
+	api "github.com/arangodb/kube-arangodb/pkg/apis/deployment/v1alpha"
+	"github.com/arangodb/kube-arangodb/pkg/client"
+)
+
+func TestLoadBalancingCursorVST(t *testing.T) {
+	// run with VST
+	LoadBalancingCursorSubtest(t, true)
+}
+
+func TestLoadBalancingCursorHTTP(t *testing.T) {
+	// run with HTTP
+	LoadBalancingCursorSubtest(t, false)
+}
+
+func wasForwarded(r *driver.Response) bool {
+	h := (*r).Header("x-arango-request-served-by")
+	return h != ""
+}
+
+// tests cursor forwarding with load-balanced conn.
+func LoadBalancingCursorSubtest(t *testing.T, useVst bool) {
```

Comment:
This function should start with a lower-case letter.
Uppercase is used to export a member outside its package, which is not needed (and wanted) here.